app/renderer/js/main.js

```diff
@@ -152,6 +154,35 @@ class ServerManagerView {
 	}
 
 	initActions() {
+		this.initDNDButton();
+		this.$dndButton.addEventListener('click', () => {
+			const dnd = ConfigUtil.getConfigItem('dnd', false);
```

Comment:
Once again this is very hard to review an trace back the where the bug is happing if the situation arises. This needs a lot of code refactoring. And this might explain why is not working for me locally.